test/_certificates/root_ca.crt

```diff
@@ -0,0 +1,34 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Would it not be better to output the certs in DER format so that they are compatible with `X509.sol`?

Reply:
As this certificate is the main Root of Trust, it isn't sent to the Smart Contract, then I didn't export it in DER format, only the end-user ones are in DER format.

Reply:
That's true, we don't actually need the certificate for nightfall to work, but it's a bit inconsistent to output it in PEM format when the others are DER.

Reply:
Yes, I'll remove this one because we don't need it.